src/cnn_feature_extractor/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+project(smqtk_cnn_feature_extractor)
+cmake_minimum_required(VERSION 2.8)
+
+find_package(Caffe REQUIRED)
```

Comment:
Caffe's config says that we need to put a call to ``find_package(OpenCV)`` before the ``fine_package(Caffe)``. Not sure why, but might as well.